sysdig/resource_sysdig_secure_rule_falco_test.go

```diff
@@ -155,6 +155,34 @@ resource "sysdig_secure_rule_falco" "kube_audit" {
 }`, name, name)
```

Comment:
instead of passing name multiple times, you can pass it once and use use formatting like `%[1]s`

Reply:
thanks. I've removed the dup args with using `%[1]s`.